expo-zustand-styled-components/.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+  "nuxt.isNuxtApp": false
+}
```

Comment:
why do we have this?

Reply:
no idea sir. will remove
I think is a vscode stunt